packages/sdk/src/Base.ts

```diff
@@ -75,8 +76,9 @@ class Base {
   private addresses = config.addresses
   private chains = config.chains
   private bonders = config.bonders
-  private fees = config.fees
-  gasPriceMultiplier: number = 0
+  fees : { [token: string]: Record<string, number>}
+  gasPriceMultiplier: number = 1
```

Comment:
What is the difference between `gasPriceMultiplier` and `destinationFeeGasPriceMultiplier`?

Reply:
`gasPriceMultiplier` is the multiplier the user specifies for when sending transactions using sdk. It's different than `destinationFeeGasPriceMultiplier` because `destinationFeeGasPriceMultiplier` is only for estimating gas for destination tx if chain is L1, or ORU